e2e/support/helpers/e2e-qa-databases-helpers.js

```diff
@@ -14,29 +14,59 @@ import { createQuestion } from "./api";
  **            QA DATABASES             **
  ******************************************/
 
-export function addMongoDatabase(name = "QA Mongo") {
+export function addMongoDatabase(displayName = "QA Mongo") {
   // https://hub.docker.com/layers/metabase/qa-databases/mongo-sample-4.4/images/sha256-8cdeaacf28c6f0a6f9fde42ce004fcc90200d706ac6afa996bdd40db78ec0305
-  addQADatabase("mongo", name, QA_MONGO_PORT);
+  addQADatabase({
+    engine: "mongo",
+    displayName,
+    port: QA_MONGO_PORT,
+  });
 }
 
-export function addPostgresDatabase(name = "QA Postgres12", writable = false) {
+export function addPostgresDatabase(
+  displayName = "QA Postgres12",
+  writable = false,
+  dbName = "postgres",
+) {
   // https://hub.docker.com/layers/metabase/qa-databases/postgres-sample-12/images/sha256-80bbef27dc52552d6dc64b52796ba356d7541e7bba172740336d7b8a64859cf8
-  addQADatabase("postgres", name, QA_POSTGRES_PORT, writable);
+  addQADatabase({
+    engine: "postgres",
+    displayName,
+    dbName,
+    port: QA_POSTGRES_PORT,
+    enable_actions: writable,
+  });
 }
 
-export function addMySQLDatabase(name = "QA MySQL8", writable = false) {
+export function addMySQLDatabase({
+  displayName = "QA MySQL8",
+  writable = false,
+}) {
   // https://hub.docker.com/layers/metabase/qa-databases/mysql-sample-8/images/sha256-df67db50379ec59ac3a437b5205871f85ab519ce8d2cdc526e9313354d00f9d4
-  addQADatabase("mysql", name, QA_MYSQL_PORT, writable);
+  addQADatabase({
+    engine: "mysql",
+    displayName: name,
+    port: QA_MYSQL_PORT,
+    enable_actions: writable,
+  });
 }
 
-function addQADatabase(engine, db_display_name, port, enable_actions = false) {
+function addQADatabase({
+  engine,
+  displayName,
+  dbName,
+  port,
+  enable_actions = false,
+}) {
   const PASS_KEY = engine === "mongo" ? "pass" : "password";
   const AUTH_DB = engine === "mongo" ? "admin" : null;
   const OPTIONS = engine === "mysql" ? "allowPublicKeyRetrieval=true" : null;
 
-  const db_name = enable_actions
-    ? WRITABLE_DB_CONFIG[engine].connection.database
-    : QA_DB_CREDENTIALS.database;
+  const db_name =
+    dbName ??
```

Comment:
this is the only bit that really matters, it allows us to pass in a dbName so we can add any arbitrary db